src/blocked.c

```diff
@@ -93,7 +93,7 @@ void blockClient(client *c, int btype) {
 
     c->flags |= CLIENT_BLOCKED;
     c->bstate.btype = btype;
-    server.blocked_clients++;
+    if (!(c->flags & CLIENT_MODULE)) server.blocked_clients++; /* We count blocked client stats on regular clients and not on module clients */
```

Comment:
just to make sure, this code path is new, right?
i.e. before this PR, module (fake) clients didn't get blocked, so that increment didn't affect them regardless of this condition.

Reply:
Yes right, before this PR we could not get to this code with module client.